pnpm-lock.yaml

```diff
@@ -1293,6 +1293,7 @@ packages:
   /@intlify/shared@11.0.1:
```

Comment:
Hmm, `pnpm-lock.yaml` has not reverted completly...
You can try again with a new PR, committing only the following changes.

```diff
diff --git a/package.json b/package.json
index a11c3f1..e75e121 100644
--- a/package.json
+++ b/package.json
@@ -59,10 +59,10 @@
     "vue": "^3.2.45"
   },
   "peerDependencies": {
-    "@intlify/shared": "^9.0.0 || ^10.0.0",
+    "@intlify/shared": "^9.0.0 || ^10.0.0 || ^11.0.0",
     "@vue/compiler-dom": "^3.0.0",
     "vue": "^3.0.0",
-    "vue-i18n": "^9.0.0 || ^10.0.0"
+    "vue-i18n": "^9.0.0 || ^10.0.0 || ^11.0.0"
   },
   "peerDependenciesMeta": {
     "@intlify/shared": {
```

Reply:
I apologize for the incomplete revert. I'll create a new PR that includes only the necessary `pnpm-lock.yaml` changes. 🙇